src/Fixer/Casing/LowercaseConstantsFixer.php

```diff
@@ -24,8 +27,29 @@
  *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-final class LowercaseConstantsFixer extends AbstractFixer
+final class LowercaseConstantsFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    /**
+     * Hold the function that will be used to convert the constants.
+     *
+     * @var callable
+     */
+    private $fixFunction;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        $functionName = 'strto'.strtolower($this->configuration['syntax']);
+
+        $this->fixFunction = function ($string) use ($functionName) {
+            return $functionName($string);
+        };
```

Comment:
I think @kubawerlos meant something like:

```
if ('upper' === $this->configuration['syntax']) {
     $this->fixFunction = static function ($string) {
          return strtoupper($string);
     };
} else {
     $this->fixFunction = static function ($string) {
          return strtolower($string);
     };
}
```

so not calling a function by using string operations to get to the name :)

Reply:
Ok I will fix it asap.

Reply:
Fixed, thanks for the insight !